src/Schema/AbstractSchema.php

```diff
@@ -422,6 +459,18 @@ protected function getColumnPhpType(ColumnSchemaInterface $column): string
         };
     }
 
+    protected function createPhpTypeColumnSchema(string $phpType, string $name): ColumnSchemaInterface
```

Comment:
It's bad name for function

Reply:
Any suggestions for a name?

Reply:
Renamed to `createColumnSchemaFromPhpType()`